libkbfs/folder_branch_ops.go

```diff
@@ -1125,11 +1125,39 @@ func (fbo *folderBranchOps) maybeUnembedAndPutBlocks(ctx context.Context,
 	return bps, nil
 }
 
+// ResetRootBlock creates a new empty dir block and sets the given
+// metadata's root block to it.
+func ResetRootBlock(ctx context.Context, config Config,
+	currentUID keybase1.UID, rmd *RootMetadata) (
+	Block, BlockInfo, ReadyBlockData, error) {
+	newDblock := NewDirBlock()
+	info, plainSize, readyBlockData, err :=
+		ReadyBlock(ctx, config, rmd.ReadOnly(), newDblock, currentUID)
+	if err != nil {
+		return nil, BlockInfo{}, ReadyBlockData{}, err
+	}
+
+	now := config.Clock().Now().UnixNano()
+	rmd.data.Dir = DirEntry{
+		BlockInfo: info,
+		EntryInfo: EntryInfo{
+			Type:  Dir,
+			Size:  uint64(plainSize),
+			Mtime: now,
+			Ctime: now,
+		},
+	}
+	co := newCreateOpForRootDir()
+	rmd.AddOp(co)
+	rmd.AddRefBlock(rmd.data.Dir.BlockInfo)
+	rmd.SetUnrefBytes(0)
```

Comment:
`MakeSuccessor` already calls `md.ClearBlockChanges` which resets this field.  I guess it doesn't hurt to do it here again, but it shouldn't be needed.  I see you copied it from the TLF init code, but I don't see why it would have been necessary there either...